eng/templates/Azure.Management.Template/src/Azure.Management.Template.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>0.1.0</Version>
+    <PackageId>Azure.Management.Template</PackageId>
+    <Description>Azure management client SDK for AzureRP </Description>
+    <PackageTags>azure;management;ProviderNameLowercase</PackageTags>
```

Comment:
I assume things like ProviderNameLowercase is just a placeholder?

Reply:
Yes, it now takes a single parameter say Microsoft.KeyVault or Microsoft.Network. The keyvault, network (lower case) will show up here.